backend/eatandmeet/api/admin.py

```diff
@@ -0,0 +1,6 @@
+from django.contrib import admin
+from .models import Event
```

Comment:
```suggestion

from .models import Event
```

One empty line between stdlib and local imports.